src/Microsoft.DotNet.VersionTools/project.json

```diff
@@ -0,0 +1,16 @@
+{
+  "dependencies": {
+    "Microsoft.Build.Framework": "0.1.0-preview-00022",
+    "Microsoft.Build.Tasks.Core": "0.1.0-preview-00022",
+    "Microsoft.Build.Utilities.Core": "0.1.0-preview-00022",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.2",
+    "NETStandard.Library": "1.6.0",
+    "Newtonsoft.Json": "7.0.1",
```

Comment:
I should have mentioned: this project.json is copied from the `Microsoft.DotNet.Build.Tasks` one, so these dependencies are mostly from there.

My changes:
* Upgraded `NETStandard.Library` to `1.6.0`
* Removed some `System.*` direct dependencies.
* Added CoreCLR runtime direct dependency. (Will discuss in other comment.)

Because of the way this is shipped (the netstandard and net45 dlls copied to the buildtools package) I wonder if changing this version might break something... I only fiddled with the project.jsons to the point where things worked, so this is something to revisit.